.github/workflows/release.yaml

```diff
@@ -80,8 +131,6 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   release-gnu:
     runs-on: ubuntu-latest
-    strategy: 
-      fail-fast: false
```

Comment:
Any reason you're removing this?

Reply:
Yes, because it is invalid syntax without the matrix keyword.